tidy3d/web/api/asynchronous.py

```diff
@@ -2,8 +2,8 @@
 from typing import Dict, List
 
 from .container import DEFAULT_DATA_DIR, BatchData, Batch
-from ..components.simulation import Simulation
-from ..log import log
+from tidy3d.components.simulation import Simulation
```

Comment:
we still want these to be relative imports. I think it just means adding another `.`  in front for example 

```
from ...log import log
```

Reply:
modified